https://issues.redhat.com/browse/ACM-13088 (#6770)
open-cluster-management/rhacm-docs: governance/gatekeeper_operator/config_gk_operator.adoc

    auditEventsInvolvedNamespace: Enabled <1>
    logLevel: DEBUG
    auditInterval: 10s
    constraintViolationLimit: 55
    auditFromCache: Enabled
    auditChunkSize: 66
    emitAuditEvents: Enabled
    resources:
      limits:
        cpu: 500m
        memory: 150Mi
      requests:
        cpu: 500m
        memory: 130Mi
  validatingWebhook: Enabled
  mutatingWebhook: Enabled
  webhook:
    replicas: 3
    emitAdmissionEvents: Enabled
    admissionEventsInvolvedNamespace: Enabled <2>
    disabledBuiltins:
     - http.send
    operations: <3>
     - "CREATE"
     - "UPDATE"
     - "CONNECT"
    failurePolicy: Fail
    resources:
      limits:
        cpu: 480m
        memory: 140Mi
      requests:
        cpu: 400m
        memory: 120Mi
  nodeSelector:
    region: "EMEA"
  affinity:
    podAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        - labelSelector:
            matchLabels:
              auditKey: "auditValue"
          topologyKey: topology.kubernetes.io/zone
  tolerations:
    - key: "Example"
      operator: "Exists"
      effect: "NoSchedule"
  podAnnotations:
    some-annotation: "this is a test"
    other-annotation: "another test"
----

> *Note:* For versions 3.14 or later, you cam implement the following features to this YAML:

Reviewer: ```suggestion
*Note:* For versions 3.14 or later, you can implement the following features from this YAML:
```

<1> Enable the `auditEventsInvolvedNamespace` parameter to manage the namespace audit event you want to create. When you enable this parameter, the Gatekeeper controller deployment gets the following argument: `--audit-events-involved-namespace: true`.

> <2> Enable the `admissionEventsInvolvedNamespace` parameter to manage the namespace admission event you want to create. When you enable this parameter, the Gatekeeper controller deployment gets the following argument: `--admission-events-involved-namespace: true`.

Reviewer: ```suggestion
<1> Enable the `auditEventsInvolvedNamespace` parameter to manage the namespace audit event you want to create. When you enable this parameter, the Gatekeeper controller deployment is run with the following argument: `--audit-events-involved-namespace=true`.
<2> Enable the `admissionEventsInvolvedNamespace` parameter to manage the namespace admission event you want to create. When you enable this parameter, the Gatekeeper controller deployment is run with the following argument: `--admission-events-involved-namespace=true`.
```

> <3> To manage your webhook operations, use the following values for the `operations` parameter, `"CREATE"`, `"UPDATE"`, `"CONNECT"`, and `"DELETE"`.

Reviewer: ```suggestion
<3> To manage your webhook operations, you can use the following values for the `operations` parameter, `"CREATE"`, `"UPDATE"`, `"CONNECT"`, and `"DELETE"`.
```

To configure the `auditFromCache` parameter for resource collection from constraints, complete the following steps: 

. Set `auditFromCache` to `Automatic` in the `Gatekeeper` resource. See the following example:

+
[source,yaml]
----
apiVersion: operator.gatekeeper.sh/v1alpha1
kind: Gatekeeper 
metadata: 
  name: gatekeeper 
spec: 
  audit:
    replicas: 2 
    logLevel: DEBUG 
    auditFromCache: Automatic
----

. To verify that the resources are added to your `Config` resource, view that the `syncOnly` parameter section is added. Run the following command:

+
[source,bash]
----
oc get configs.config.gatekeeper.sh config -n openshift-gatekeeper-system
----
+
Your `Config` resource might resemble the following example:

+
[source,yaml]
----
apiVersion: config.gatekeeper.sh/v1alpha1
kind: Config
metadata:
 name: config
 namespace: "openshift-gatekeeper-system"
spec:
 sync:
   syncOnly:
   - group: ""
     version: "v1"
     kind: "Namespace"
   - group: ""
     version: "v1"
     kind: "Pod"
----

> . Get the explanation of the `auditFromCache` setting to view the description of the custom resource by running the following command:

Reviewer: (Optional) This is more of a note than an additional step in the process (though it's also fine as it is).
```suggestion
**NOTE:** You can view the explanation of the `auditFromCache` setting from the description of the {gate} custom resource by running the following command:
```

Reviewer 2: We do not need to use double asterisk to bold. Reminder for @jc-berger we also do not capitalize note. It seems like it would be more appropriate to use "*Optional:*"

> +
> [source,bash]
> ----
> oc get configs.config.gatekeeper.sh config -n openshift-gatekeeper-system
> ----
> +
> Your `Config` resource might resemble the following example:
> 
> +
> [source,yaml]
> ----
> apiVersion: config.gatekeeper.sh/v1alpha1
> kind: Config
> metadata:
>  name: config
>  namespace: "openshift-gatekeeper-system"
> spec:
>  sync:
>    syncOnly:
>    - group: ""
>      version: "v1"
>      kind: "Namespace"
>    - group: ""
>      version: "v1"
>      kind: "Pod"
> ----
> 
> **NOTE:** You can view the explanation of the `auditFromCache` setting from the description of the {gate} custom resource by running the following command:

Reviewer 2: ```suggestion
+
[source,bash]
----
oc get configs.config.gatekeeper.sh config -n openshift-gatekeeper-system
----
+
Your `Config` resource might resemble the following example:

+
[source,yaml]
----
apiVersion: config.gatekeeper.sh/v1alpha1
kind: Config
metadata:
 name: config
 namespace: "openshift-gatekeeper-system"
spec:
 sync:
   syncOnly:
   - group: ""
     version: "v1"
     kind: "Namespace"
   - group: ""
     version: "v1"
     kind: "Pod"
----
+
*Note:* You can view the explanation of the `auditFromCache` setting from the description of the {gate} custom resource by running the following command:
```

> *Optional* You can view the explanation of the `auditFromCache` setting from the description of the {gate} custom resource by running the following command:
> 
> +
> [source,bash]
> ----

Reviewer: Missing colon and has an extra `+` (I _think_ the plus is no longer necessary since it's no longer in the list?)
```suggestion
*Optional:* You can view the explanation of the `auditFromCache` setting from the description of the {gate} custom resource by running the following command:

[source,bash]
----
```
